Detections/MultipleDataSources/HostAADCorrelation.yaml

```diff
@@ -34,13 +37,25 @@ query: |
   let signin_threshold = 5; 
   //Make a list of IPs with failed Windows host logins above threshold
   let win_fails = 
-  SecurityEvent
+  (union isfuzzy=true
```

Comment:
I don't think union will work here. You need to cerate a single list, and the union will create two lists and will fail in line 72. The easiest method is to create two separate sub-queries  (win_fails, wef_fails or other names) and check both in line 72 as the query currently do for Unix and Windows.